assets/js/components/Dashboards/PreservationChecks/List.jsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import { useQuery, useLazyQuery } from "@apollo/client";
+import { GET_PRESERVATION_CHECKS } from "@js/components/Dashboards/dashboards.gql";
+import UIDate from "@js/components/UI/Date";
+import IconDownload from "@js/components/Icon/Download";
+import { GET_PRESIGNED_URL } from "@js/components/IngestSheet/ingestSheet.gql.js";
+
+const colHeaders = ["Started", "Completed", "Status", "Errors", "Filename", ""];
+
+function Status({ status }) {
+  let statusClass = "";
+
+  switch (status) {
+    case "COMPLETE":
+      statusClass = "is-success";
+      break;
+    case "ERROR":
+      statusClass = "is-danger";
+      break;
+    case "TIMEOUT":
+      statusClass = "is-danger";
+      break;
+    case "ACTIVE":
+      statusClass = "is-warning";
+      break;
+    default:
+      break;
+  }
+
+  return <span className={`tag ${statusClass} is-light`}>{status}</span>;
```

Comment:
We now have a `Tag` component you can import:  https://nulib.github.io/admin-react-components/styleguide/#tag